src/app/workshop-details/workshop-details.component.html

```diff
@@ -31,21 +32,22 @@
             <div id="{{tabs[2].label}}" [hidden]="!tabs[2].showTab" class="tabcontent">
                 <div class="photographer-information" *ngFor="let photographer of workshopDetails.photographers;">
                     <div class="profile-photo">
-                        <img class="profile-img" [src]="this.workshopRepository.globalConstants.resolveImageUrl(photographer.profilePhotoLink)" alt="workshop image"
-                        />
+                        <object class="profile-img" [data]="this.workshopRepository.globalConstants.resolveLocalImageUrl(photographer.profilePhotoLink)" type="image/png" alt="workshop image">
+                            <img class="profile-img" [src]="this.workshopRepository.globalConstants.resolveImageUrl(photographer.profilePhotoLink)" alt="workshop image"/>
+                        </object>
                     </div>
                     <h4>
                         <span>{{photographer.firstName}}</span>
                         <span>{{photographer.lastName}}</span>
                     </h4>
-                    <div class="detail-location">
+                    <div class="detail-location" *ngIf="photographer.locationName !== null && photographer.locationName !== ''">
```

Comment:
what if it's undefined? I think photographer.locationName is enough instead of null

Reply:
it'll actually always be empty due to json serialization and never null or undefined.